utility/utils.py

```diff
@@ -609,6 +609,31 @@ def get_latest_container(version):
     }
 
 
+def get_release_repo(version):
+    """
+    Retrieves the repo and image information for the RC build of the version specified from magna002.ceph.redhat.com
+
+    Args:
+        version: version to get the latest image tag, should match version in release.yaml at magna002
+                 storage
+
+    Returns:
+        Repo and Container details dictionary with given format:
+        {'composes': <RC release composes>, 'image': <CEPH and monitoring images related to the RC build>}"""
+    url = "http://magna002.ceph.redhat.com/cephci-jenkins/latest-rhceph-container-info/release.yaml"
+    data = requests.get(url, verify=False)
+    repo_details = yaml.safe_load(data.text)[version]
+    return repo_details
+
+
+def load_file(file_name):
```

Comment:
can we rename this method to `yaml_to_dict` | `load_yaml_file`?

Reply:
Done.